openapi/components/parameters/journalRecordId.yaml

```
@@ -0,0 +1,8 @@
+name: journalRecordId
+in: path
+description: Journal record identifier string.
```

nitpick: based on our style
```suggestion
description: ID of the journal record.
```